### PR TITLE
Turn off gender icons in chat

### PR DESCRIPTION
**Pull request by AerysBat**
Suggested during a discussion in the slimcat channel.

### CODE REVIEW

**Comment by WreckedAvent** [comment]
Could you please open issues before you make pull requests so if there's any chance there's disagreement they can be discussed? I'd hate for you to do any more work and get rebuffed.

I am not too interested in a gender icon toggle. slimCat is designed to be used with minimal or no gender coloring, and I do not require themes to support full gender coloring. The full coloration is provided as convenience for web-client users attempting to get used to slimCat.

**Comment by AerysBat** [comment]
* I do not think I explained what this does very well. This does not turn off all gender icons. The option is to turn off the gender icons in the chat area. Gender icons in the sidebar are unaffected.
* I like enabling this option for two reasons.  First, most of the time in channels I frequent the same people lead the conversation, and I do not need reminders of their characters' gender.  Second, even though the size of the icon is matched to the size of the text, it is still slightly obtrusive a text-only environment.  The conversation I mentioned earlier hit me in a "cannot unsee" kind of way.

**Comment by WreckedAvent** [comment]
Thought about this for a while.

I still do not like the idea of hiding the gender icons (anywhere), BUT, this is something that adds value for some people while not removing any value for most slimCat users, or in anyway impacting their experience.

File: slimCat/Views/UI Parts/ChannelMessageView.xaml [hunk 1]
```diff
@@ -35,7 +36,8 @@
 
     <InlineUIContainer BaselineAlignment="Bottom">
         <Rectangle Height="{Binding Source={x:Static models:ApplicationSettings.FontSize}}" Width="{Binding Source={x:Static models:ApplicationSettings.FontSize}}"
-                   Fill="{Binding Path=Poster, Converter={StaticResource GenderColorConverter}, Mode=OneWay}">
+                   Fill="{Binding Path=Poster, Converter={StaticResource GenderColorConverter}, Mode=OneWay}"
+                   Visibility="{Binding Source={x:Static models:ApplicationSettings.ShowGenderIcons}, Converter={StaticResource GreaterThanZeroConverter}}">
```
Comment (WreckedAvent) [inline slimCat/Views/UI Parts/ChannelMessageView.xaml]: `GreaterThanZeroConverter` is not a typo? It seems like you want `BoolConverter` here

**Comment by AerysBat** [comment]
Thank you for understanding. 
I was not aware of the BoolConverter class, it is certainly more appropriate.

**Comment by WreckedAvent** [comment]
:100: approved.

**Pull request merged**